uds/database/data_record/mapping_data_record.py

```
@@ -1,72 +1,103 @@
+        :param raw_value: Raw (bit) value of this Data Record single occurrence.
 
-        :return: Dictionary with physical value for this Data Record.
+        :return: Decoded physical (a label) value for this occurrence.
         """
-        if raw_value in self.mapping:
-            physical_value = self.mapping[raw_value]
-            return DecodedDataRecord(name=self.name, raw_value=raw_value, physical_value=physical_value)
-        return DecodedDataRecord(name=self.name, raw_value=raw_value, physical_value=raw_value)
+        if raw_value in self.values_mapping:
+            return self.values_mapping[raw_value]
+        warn(message="There is no label defined in mapping for provided value.",
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)